charts/vsphere-cpi/templates/configmap.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: vsphere-cloud-config
+  name: {{ .Values.config.name }}
```

Comment:
Since we have cloud-config as the default name for configmap, could you also add it here?

Reply:
Just added static inline defaults in secret.yaml and configmap.yaml. Thanks!